backend/app/usecase/shortlink/metatag.go

```diff
@@ -0,0 +1,81 @@
+package shortlink
+
+import (
+	"github.com/short-d/short/backend/app/entity/metatag"
+	"github.com/short-d/short/backend/app/usecase/repository"
+)
+
+var _ MetaTag = (*MetaTagPersist)(nil)
+
+// MetaTag represents a ShortLink MetaTags retriever and updater
+type MetaTag interface {
+	GetOpenGraphTags(alias string) (metatag.OpenGraph, error)
+	GetTwitterTags(alias string) (metatag.Twitter, error)
+}
+
+// MetaTagPersist represents a ShortLink MetaTags retriever and updater which persist the generated
+// changes in the repository
+type MetaTagPersist struct {
+	shortLinkRepo repository.ShortLink
+}
+
+const defaultTitle string = "Short: Free link shortening service"
+const defaultDesc string = "Short enables people to type less for their favorite web sites"
+const defaultImageURL string = "https://short-d.com/promo/small-tile.png"
+
+// GetOpenGraphTags retrieves ShortLink Open Graph Tags from persistent storage given alias
+func (m MetaTagPersist) GetOpenGraphTags(alias string) (metatag.OpenGraph, error) {
+	shortLink, err := m.shortLinkRepo.GetShortLinkByAlias(alias)
+	if err != nil {
+		return metatag.OpenGraph{}, err
+	}
+
+	defaultTitle := defaultTitle
+	defaultDesc := defaultDesc
+	defaultImageURL := defaultImageURL
```

Comment:
Those variables are needed only if the short link corresponding field is empty. Therefore you can move them there.